### DISCUSSION TITLE
Running from Jupyternote

### DISCUSSION THREAD

**Discussion by him1532** [SYSTRAN/faster-whisper#286] [Q&A] [answered]
I am trying to run your example in jupyternotebook. 
I've installed all the relevant library/module

My first question is  when I try to import the WhipserModel 
I get 
`ModuleNotFoundError: No module named 'faster_whisper'`

When I run this on a terminal this step is fine. Is there a way to solve this ?

My second question is regarding the compute_type parameter. 
When I try the  float16 I get the following error
`ValueError: Requested float16 compute type, but the target device or backend do not support efficient float16computation.`

I get the same error for int8_float16.  So how do you know which is proper parameter before you run into error?
Does this mean I can't use cuda? I have the cuda library installed. 

Thank you.

**Comment by guillaumekln** [comment, marked as answer]
I'm not familiar with Jupyter Notebook so I can't really help for the installation issue in this environment.

Regarding the error with `compute_type`, it means your GPU does not support float16 execution. You can either remove the argument entirely or set other compute types such as `compute_type="float32"` or `compute_type="int8"`.

**Reply by otakutyrant** [reply]
OMG! My computer can run whisper with small.en well but faster-whisper cannot work with the same model for me until I change the `compute_type` from `float16` to `float32`.
